build-system/tasks/runtime-test/helpers.js

```diff
@@ -310,6 +311,13 @@ async function runTestInBatches_(config, browsers) {
       totalStableFailed += results.failed;
     }
   };
+  const betaRunCompleteFn = async (browsers, results) => {
```

Comment:
This is a duplicate of https://github.com/ampproject/amphtml/blob/fc694931251570f4ea4355567ec5ca352129add4/build-system/tasks/report-test-status.js#L161-L167